src/protos/io/defang/v1/fabric.proto

```diff
@@ -371,10 +369,12 @@ message Event {
 message PublishRequest { Event event = 1; }
 
 message SubscribeRequest { 
-  repeated string services = 1; 
+  repeated string services = 1;
 }
 
-message SubscribeResponse { repeated ServiceInfo services = 1; }
+message SubscribeResponse { 
```

Comment:
Same thing applies here: because these are server-side streaming rpcs, the client will get many responses, so we only need to consider what a single response looks like.